server/src/main/java/dev/slimevr/config/DriftCompensationConfig.java

```diff
@@ -33,6 +33,15 @@ public void updateTrackersDriftCompensation() {
 		}
 	}
 
+	public void clearTrackersDriftCompensation() {
+		for (Tracker t : Main.getVrServer().getAllTrackers()) {
+			Tracker tracker = t.get();
+			if (tracker instanceof IMUTracker imuTracker) {
+				imuTracker.clearDriftCompensation();
+			}
+		}
+	}
```

Comment:
It feels weird to be introducing non-config related functionality in the config classes... Is there a better place this can be put?

Reply:
Moved it to vrserver